backend/src/routes/api/namespaces/namespaceUtils.ts

```diff
@@ -16,6 +18,36 @@ export const applyNamespaceChange = (
       400,
     );
   }
+  const kc = fastify.kube.config;
+  const cluster = kc.getCurrentCluster();
+
+  const selfSubjectAccessReviewResult = await passThrough<V1SelfSubjectAccessReview>(
+    fastify,
+    request,
+    {
+      url: `${cluster.server}/apis/authorization.k8s.io/v1/selfsubjectaccessreviews`,
+      method: 'POST',
+      requestData: JSON.stringify({
```

Comment:
I'm a little disappointed this object is not typed.

Can you make this whole call a new function, provide a type to the data you are giving to `requestData`. 

It'll help with keeping this specific block of code out of the namespaceUtils directly, giving us a more declarative approach to this code?